doc/testing_tools/testing_floating_points.qbk

```diff
@@ -135,29 +140,42 @@ inequations [link equ2 (2)] and [link equ3 (3)] where all underflow, overflow co
 || abs(u - v)/abs(v) <= epsilon; // (5)
 ``
 
+The above however, will not work when `v` or `u` is too close to zero. In such cases the solution is to resort to [link equ1 (1)]. 
+
 [h3 Implementation]
 
-Checks based on equations [link equ4 (4)] and [link equ5 (5)] is implemented by the binary predicate 
-[classref boost::math::fpc::close_at_tolerance `close_at_tolerance`]. The method for comparison (Equ [link equ4 (4)] or [link equ5 (5)]) as well as the
-epsilon are selected at construction. This predicate reports the tolerance to set to make the test successful.
+Binary predicate [classref boost::math::fpc::close_at_tolerance `close_at_tolerance`] implements checks [link equ4 (4)] and [link equ5 (5)].
+The tolerance and the seleciotn between [link equ4 (4)] and [link equ5 (5)] is determined during construction. Apart from the Boolean result,
+the predicate can also return the minimum tolerance required to check positively the closeness of `u` and `v`.
```

Comment:
"the minimum tolerance required to check positively ": this is incorrect and this is not what "the tolerance to set to make the test successful" says:
- the tolerance to set depends on the equation currently under check
- it also handles the inequality, while you are saying "check positively the closeness of `u` and `v`"